Source/Scene/ImplicitTileCoordinates.js

```diff
@@ -178,12 +394,128 @@ ImplicitTileCoordinates.prototype.deriveChildCoordinates = function (
   // Quadtree
   return new ImplicitTileCoordinates({
     subdivisionScheme: this.subdivisionScheme,
+    subtreeLevels: this.subtreeLevels,
     level: level,
     x: x,
     y: y,
   });
 };
 
+/**
+ * Get the coordinates of the subtree that contains this tile. If the tile is
+ * the root of the subtree, the root of the subtree is returned.
+ *
+ * @returns {ImplicitTileCoordinates} The subtree that contains this tile
+ * @private
+ */
+ImplicitTileCoordinates.prototype.getSubtreeCoordinates = function () {
+  return this.getAncestorCoordinates(this.level % this.subtreeLevels);
+};
+
+/**
+ * Get the coordinates of the parent subtree that contains this tile
+ *
+ * @returns {ImplicitTileCoordinates} The parent subtree that contains this tile
+ * @private
+ */
+ImplicitTileCoordinates.prototype.getParentSubtreeCoordinates = function () {
+  return this.getAncestorCoordinates(
+    (this.level % this.subtreeLevels) + this.subtreeLevels
+  );
+};
+
+/**
+ * Returns whether this tile is an ancestor of another tile
+ *
+ * @param {ImplicitTileCoordinates} descendantCoordinates the descendant coordinates
+ * @returns {Boolean} <code>true</code> if this tile is an ancestor of the other tile
+ * @private
+ */
+ImplicitTileCoordinates.prototype.isAncestor = function (
+  descendantCoordinates
+) {
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("descendantCoordinates", descendantCoordinates);
+  checkMatchingProperties(this, descendantCoordinates);
+  //>>includeEnd('debug');
+
+  var subdivisionScheme = this.subdivisionScheme;
+  var tileLevel = this.level;
+  var tileX = this.x;
+  var tileY = this.y;
+
+  var levelDifference = descendantCoordinates.level - tileLevel;
+  if (levelDifference <= 0) {
+    return false;
+  }
+
+  var isAncestorX = tileX === descendantCoordinates.x >> levelDifference;
+  var isAncestorY = tileY === descendantCoordinates.y >> levelDifference;
```

Comment:
Minor nitpick, I feel like this would be a little easier to understand if we didn't combine the shift and the `===` in one line, what do you think?

```suggestion
  var ancestorX = descendantCoordinates.x >> levelDifference;
  var ancestorY = descendantCoordinates.y >> levelDifference;
  ...
  return tileX == ancestorX && tileY == ancestorY;
```